apps/main/src/llamalend/features/user-position-history/cells/TimestampCell.tsx

```diff
@@ -0,0 +1,14 @@
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import type { CellContext } from '@tanstack/react-table'
+import { formatDate } from '@ui/utils'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import type { ParsedUserLendCollateralEvent } from '../hooks/useUserLendCollateralEvents'
+
+const { Spacing } = SizesAndSpaces
+
+export const TimestampCell = ({ row }: CellContext<ParsedUserLendCollateralEvent, any>) => (
+  <Stack paddingTop={Spacing.sm} paddingBottom={Spacing.sm} paddingRight={Spacing.sm}>
+    <Typography variant="tableCellSBold">{formatDate(row.original.timestamp, 'long')}</Typography>
```

Comment:
With this little change your cell is not 100% reusable for multiple columns
```suggestion
    <Typography variant="tableCellSBold">{formatDate(row.getValue(), 'long')}</Typography>
```